x-pack/plugins/enterprise_search/server/ui_settings.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { UiSettingsParams } from '@kbn/core/types';
+import { i18n } from '@kbn/i18n';
+
+import { enterpriseSearchFeatureId, enableIndexPowerupsTab } from '../common/ui_settings_keys';
+
+/**
+ * uiSettings definitions for Enterprise Search
+ */
+export const uiSettings: Record<string, UiSettingsParams<boolean>> = {
+  [enableIndexPowerupsTab]: {
+    category: [enterpriseSearchFeatureId],
+    description: i18n.translate('xpack.enterpriseSearch.enableIndexPowerupsTab.description', {
```

Comment:
nitpick: the label key being more expressive? `xpack.enterpriseSearch.uiSettings.indexPowerups.description` ?

Reply:
I like that, it'll also make `powerUps` easier to replace as I'm not sure that naming will make it to release.